erc/ERCS/erc-7562.md

```diff
@@ -348,11 +353,11 @@ These are storage rules that help to protect the bundler from denial of service:
     * **`opsSeen`, `opsIncluded`, and reputation calculation** are defined above.
     * `UnstakedReputation` of an entity determines the maximum number of entries using this entity allowed in the mempool.
     * `opsAllowed` is a reputation-based calculation for an unstaked entity, representing how many `UserOperations` it is allowed to have in the mempool.
-* **[UREP-010]** An unstaked sender is only allowed to have `SAME_SENDER_MEMPOOL_COUNT` `UserOperation`s  in the mempool. A staked sender is only limited by the SREP rules.
-* **[UREP-020]** For other entities: \
-  `opsAllowed = SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT + (inclusionRate * INCLUSION_RATE_FACTOR) + (min(opsIncluded, 10000)`.
-    * This is a default of `SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT` for a new entity
-* **[UREP-030]** If an unstaked entity causes an invalidation of a bundle, its `opsSeen` is set to `1000`, effectively blocking it from inclusion for 24 hours.
+  * Rules:
+    * **[UREP-010]** An unstaked sender is only allowed to have `SAME_SENDER_MEMPOOL_COUNT` `UserOperation`s  in the mempool. A staked sender is only limited by the SREP rules.
+    * **[UREP-020]** For other unstaked `OK` entities: \
+      `opsAllowed = SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT + inclusionRate * (INCLUSION_RATE_FACTOR + min(opsIncluded, 10000))`.
```

Comment:
1. Consider whether INCLUSION_RATE_FACTOR is needed at all.  It just starts the slow-start at 20 instead of 10 and does nothing else.
2. Make both 10000's constants.  We have them in two unrelated places.  One is the max ops allowed for an unstaked entity, the other (which appears in several rules) is used to trigger banning by setting opsSeen=10000. For the latter, specify the number of hours it takes to unban with this value.

Reply:
Fixed and fixed